src/__experimental__/components/switch/switch-slider.style.js

```diff
@@ -0,0 +1,90 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../../style/themes/base';
+import SwitchSliderPanel from './switch-slider-panel.style';
+import ClassicSwitchSliderStyles from './switch-slider-classic.style';
+
+const StyledSwitchSlider = styled.span`
+  ${({
+    checked, disabled, size, theme
+  }) => css`
+    background-color: ${theme.switch.off};
```

Comment:
Double-check styles here... Don't think you need the `right` and `bottom` as you have a `width` and `height`.

Reply:
👍 addressed